templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,293 @@
 # Import from Slack
 
-Starting with Zulip 1.8, Zulip supports importing data from Slack,
-including users, channels, messages, attachments, avatars, custom
-emoji, and emoji reactions.
+You can import your Slack organization into Zulip. It's a great way to preserve
+your organization's history when you migrate from Slack to Zulip, and to make
+the transition easy for the members of your organization.
 
-This tool has been used to import Slack workspaces with 10,000 members
-and millions of messages. If you're planning on doing an import much
-larger than that, or run into performance issues when importing,
-[contact us](/help/contact-support) for help.
+The import will include:
 
-**Note:** You can only import a Slack workspace as a new Zulip
```

Comment:
This NOTE is super important. If fact, I think it's too understated as it is. If someone starts using Zulip and then decides to do a Slack import, we have to torch what they did (or they have two orgs). If someone has multiple Slack imports, they get multiple Zulip orgs. It's the number 2 gripe I hear, and the only one that we could control (Number 1 is Slack doesn't export PMs)

Reply:
This is now outside the tabs in a way that I think is pretty clear.